xapian-applications/omega/datevalue.cc

```diff
@@ -30,6 +30,7 @@
 
 #include "timegm.h"
 #include "values.h"
+#include "parseint.h"
```

Comment:
In general, when adding something to a list in the code which is currently in sorted order you should add the new entry to preserve that property.